### PR TITLE
Feat/banner summit

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Banner for the Summit
Internal reference: MANAGER-1364

The code has been optimized for copy-pasting between universes.

### CODE REVIEW

File: client/app/core/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
    <translation id="universe_univers-sunrise_name" qtlid="232401">Sunrise</translation>
-<translation id="universe_univers-portal_name" translate="none">MyOVH</translation>
+   <translation id="universe_univers-portal_name" translate="none">MyOVH</translation>
    <translation id="universe_univers-partners_name" qtlid="383512">Partenaires</translation>
    <translation id="universe_univers-labs_name" qtlid="383707">UX Labs</translation>
    <translation id="universe_labs_chatbot" qtlid="382758">Assistant virtuel OVH</translation>
    <translation id="older_interface" qtlid="135931">Ancienne interface client</translation>
-   <translation id="universe_univers-v3_name" qtlid="135931">Ancienne interface client</translation>
+
+   <translation id="configuration_summit_banner_alt">M'inscrire au OVH SUMMIT PARIS qui se déroule le 18 octobre 2018 (s'ouvre dans un nouvel onglet). Retrouvez les experts OVH, nos partenaires et des startupstoujours plus innovantes.</translation>
```
Comment (marie-j) [inline client/app/core/translations/Messages_fr_FR.xml]: `startupstoujours` (Have `content` validated this ? 😄 )

Comment (JeremyDec) [inline client/app/core/translations/Messages_fr_FR.xml]: `startupstoujours` => `startups toujours`

File: client/app/configuration/configuration.html [hunk 2]
```diff
@@ -1,37 +1,53 @@
...
-        </h1>
-        <p class="lead"
-           data-ng-bind-html="'core_top_guides_header' | translate: { t0: ConfigurationCtrl.allGuides }"></p>
-    </div>
+<div id="main-container" data-ng-controller="UserCtrl">
+    <a
+        data-ng-href="https://summit.ovh.com/index.php/{{:: ConfigurationCtrl.localeForSummitBanner }}/subscription"
+        target="_blank"
+        rel="noopener">
+        <img
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/configuration/configuration.html]: This is done

File: client/app/configuration/configuration.html [hunk 3]
```diff
@@ -1,37 +1,53 @@
...
-    <div class="text-center mt-5 py-5">
-        <h1 class="d-inline">
-            <span data-translate="welcome_to_your_manager_1"></span>
-            <span data-translate="welcome_to_your_manager_2"></span>
-        </h1>
-        <p class="lead"
-           data-ng-bind-html="'core_top_guides_header' | translate: { t0: ConfigurationCtrl.allGuides }"></p>
-    </div>
+<div id="main-container" data-ng-controller="UserCtrl">
+    <a
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply client/app/configuration/configuration.html]: This is done

**Comment by ovh-cds** [comment]
CDS Report build#4479.0 ✘  
* init
  * deps ✘

**Comment by ovh-cds** [comment]
CDS Report build#4480.0 ✘  
* init
  * deps ✘

**Pull request closed without merging**